tardis/util.py

```diff
@@ -428,4 +430,14 @@ def quantity_linspace(start, stop, num, **kwargs):
         raise ValueError('Both start and stop need to be quantities with a '
                          'unit attribute')
 
-    return np.linspace(start.value, stop.to(start.unit).value, num, **kwargs) * start.unit
\ No newline at end of file
+    return np.linspace(start.value, stop.to(start.unit).value, num, **kwargs) * start.unit
+
+
+def convert_abundances_format(fname, delimiter='\s+'):
+    df = pd.read_csv(fname, delimiter=delimiter, comment='#', header=None)
+    #Drop shell index column
+    df.drop(df.columns[0], axis=1, inplace=True)
+    #Assign header row
+    df.columns = [nucname.name(i)
+                  for i in range(1, df.shape[1] + 1)]
```

Comment:
Yes, it will throw a `Runtime Error` with the wrong name of element/isotope. Ex- `RuntimeError: Not a Nuclide! XYZ --> 0`, where `XYZ` is wrongly specified element.